cmd/metrics.go

```diff
@@ -94,6 +106,26 @@ func (c *minioCollector) Collect(ch chan<- prometheus.Metric) {
 	healingMetricsPrometheus(ch)
 }
 
+func nodeHealthMetricsPrometheus(ch chan<- prometheus.Metric) {
+	nodesUp, nodesDown := GetPeerOnlineCount()
+	ch <- prometheus.MustNewConstMetric(
+		prometheus.NewDesc(
+			prometheus.BuildFQName(minioNamespace, "nodes", "online"),
```

Comment:
`minio_nodes_online` should show 4 as per the cluster level metrics understanding.

![image](https://user-images.githubusercontent.com/5156139/103202054-6b683900-4917-11eb-8154-da657c087118.png)